src/main/resources/templates/email/programme-created/v1.0.0.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org" xmlns="http://www.w3.org/1999/html">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+</head>
+<body>
+<h1>Email Message</h1>
+<h2>Subject</h2>
+<th:block th:fragment="subject">NHS England - <span
+    th:text="${not #strings.isEmpty(localOfficeName)} ? ${localOfficeName} : _" th:remove="tag"></span
+  > - Onboarding to your Training programme</th:block>
+<h2>Content</h2>
+<th:block th:fragment="content">
+  <div th:replace="fragments/greeting/v1.0.0 :: body"></div>
+  <p>Congratulations on your appointment to the <b><span
+      th:text="${not #strings.isEmpty(programmeName)} ? ${programmeName} : 'programme'"></span></b>.</p>
+  <p>Please take some time to read this email in its entirety. It contains vital information to help you start the onboarding processes for your training programme and there are some actions that you will need to do straight away.</p>
+  <p><b>Please note:</b></p>
+  <p><ul>
+  <li>We are in process of ensuring your Training Programme Director(s) (TPDs) have your details.</li>
+  <li>also be aware NHS England is not your employer, we act on behalf of your Postgraduate Dean and manage your training programme for the duration of your training.</li>
+</ul>
+  </p>
+  <p><b>Timeline:</b></p>
+  <p>Over the next couple of months, you will receive a series of automated messages from NHS England via TSS (TIS Self Service) and email communications from colleagues within <span th:text="${not #strings.isEmpty(localOfficeName)} ? ${localOfficeName} : 'your local deanery office'"></span
```

Comment:
Should be "TIS Self-Service" rather than "TIS Self Service", there are a few other instances that need changing too.

Did we previously get a request to change something worded like "local deanery office" to just be "local office"?

Reply:
Parked with other text changes to handle as a batch. I am not sure about whether 'deanery' should be included - for some reason I thought we had been asked to include it so local office -> local deanery office 😖

Reply:
iirc Naz asked us to include it, but when we demoed it in review we were asked to remove it 🤷🏻